File: app/src/main/java/com/geridea/trentastico/network/controllers/listener/LibraryOpeningTimesListener.java
package com.geridea.trentastico.network.controllers.listener;


/*
 * Created with ♥ by Slava on 28/04/2017.
 */

import com.geridea.trentastico.model.LibraryOpeningTimes;

import java.util.Calendar;

public interface LibraryOpeningTimesListener {

    void onOpeningTimesLoaded(LibraryOpeningTimes times, Calendar date);

    void onOpeningTimesLoadingError();

    void onErrorParsingResponse(Exception e);
}

File: app/src/main/java/com/geridea/trentastico/network/controllers/listener/LibraryOpeningTimesListener.kt
package com.geridea.trentastico.network.controllers.listener


/*
 * Created with ♥ by Slava on 28/04/2017.
 */

import com.geridea.trentastico.model.LibraryOpeningTimes

import java.util.Calendar

interface LibraryOpeningTimesListener {

    fun onOpeningTimesLoaded(times: LibraryOpeningTimes, date: Calendar)

    fun onOpeningTimesLoadingError()

    fun onErrorParsingResponse(e: Exception)
}
